rcudnn/cudnn/src/api/rnn.rs

```diff
@@ -205,8 +220,7 @@ impl API {
         algorithm: cudnnRNNAlgo_t,
         data_type: cudnnDataType_t,
     ) -> Result<(), Error> {
-
-        match cudnnSetRNNDescriptor(
+        match cudnnSetRNNDescriptor_v6(
```

Comment:
is this the latest or is that a descriptor that is already phased out, maybe you know a bit more about the versioning scheme of the cudnn API?

Reply:
Latest version, nothing hugely exciting, just changes the int to const int for hidden & numLayers.